src/repl.ts

```diff
@@ -230,6 +231,19 @@ function enterPasteMode() {
   })
 }
 
+function loadFile(filename: string) {
+  try {
+    console.log(`// loading file ${filename}`.cyan)
+    let file_contents: string = fs.readFileSync(filename, 'utf8')
+    console.log(colorize(file_contents))
```

Comment:
I thought verbose mode implied printing out transpiled **javascript**.
Also, printing out the entire file is whate node does on **.load**.